### PR TITLE
Added RS Zerolog logging adapter

### PR DESCRIPTION
**Pull request by matelang**
I have added support for [Zerolog](https://github.com/rs/zerolog), which is a popular and efficient zero allocation JSON logger (see benchmarks [here](https://github.com/rs/zerolog#benchmarks)).

We are using Zerolog at our organisation and we'd like to benefit of having a single logging library manage all our logging while using VMware Go KCL.

I tried implementing the logger similarly to the existing zap adapter, with the difference that in ZeroLog there is only a singleton timestamp function, so the config.LocalTime flag can not be respected without potentially breaking expectation of the user in non-library code.

Feedback is welcome and I am happy to adapt in a way that is acceptable to the maintainer.

Thanks.

### CODE REVIEW

File: logger/zerolog/zerolog_test.go [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+	contextLogger.Debugf("Starting with rs zerolog")
+	contextLogger.Infof("Rs zerolog is awesome")
+}
+
+func TestZapLogger(t *testing.T) {
+	log := NewZerologLogger()
+
+	contextLogger := log.WithFields(logger.Fields{"key1": "value1"})
+	contextLogger.Debugf("Starting with zap")
+	contextLogger.Infof("Zap is awesome")
```
Comment (taojwmware) [inline logger/zerolog/zerolog_test.go:31]: zerolog

File: logger/zerolog/zerolog.go [hunk 2]
```diff
@@ -0,0 +1,128 @@
+package zerolog
```
Comment (taojwmware) [inline logger/zerolog/zerolog.go:1]: please add a copyright header like other logger implementations.

**Comment by taojwmware** [comment]
looks good. Thanks for working on this.

File: logger/zerolog/zerolog.go (hunk 2 above)
Comment (matelang) [reply logger/zerolog/zerolog.go:23]: Fixed by 9a712eea9ac3651f24536aede5ff704f5716e862

**Pull request merged**